src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -3646,7 +3645,7 @@ namespace ConvectionCoefficients {
                 thisWWR = -999.0; // throw error?
             }
             // first pass thru this zones surfaces to gather data
-            for (SurfLoop = Zone(ZoneLoop).SurfaceFirst; SurfLoop <= Zone(ZoneLoop).SurfaceLast; ++SurfLoop) {
+            for (int SurfLoop = Zone(ZoneLoop).SurfaceFirst; SurfLoop <= Zone(ZoneLoop).SurfaceLast; ++SurfLoop) {
```

Comment:
I very much like the localization of this index variable.  Range based loops are always nice, but we lose that when we aren't looping over the full array, which is acceptable.